week-1-practise/level-1/8-function-output-1.js

```diff
@@ -6,7 +6,7 @@ var result = getNumber();
 console.log(result);
 
 // Answer these questions:
-// - What does this program do?
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+// - What does this program do? get a random number between 0 and 1; return random number times by 10.
+// - How many parameters does the function take? 2
```

Comment:
Parameters are values that are given to the function _inside_ the `()`.  Is there anything inside the parentheses of `getNumber()`?